### DISCUSSION TITLE
Management: How to track and get notifications for my posts without manual URL checks

### DISCUSSION THREAD

**Discussion by 12shida** [community/community#143877] [New to GitHub]
GitHub has so many powerful features that I'm feeling a bit lost trying to find what should be a simple option. After spending around 40 minutes exploring different sections, I still can't figure out how to track my own discussion posts efficiently.
Currently, I'm just saving URLs and checking them manually for responses. I'm sure there's a proper way to:

View all my posts in one place
Get notifications for responses
Track my discussions without manual checks

While I appreciate GitHub's comprehensive toolkit, I could use some guidance on this specific feature that I'm probably overlooking among all the available options

**Comment by Crypto-git-dev** [comment]
You should do enabling Notifications for responses
First you should "**Go to Settings > Notifications"** and ensure you have enabled notifications for discussions.
For each discussion post you participate in, click the "**Subscribe**" button (located in the top-right corner of the discussion).
This will notify you of any updates, replies, or mentions.

Best Regards

**Reply by ghostinhershell** [reply]
Hi @12shida 👋🏾,It looks like @Crypto-git-dev's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

If not, can you follow-up with more information?
